joomla/joomla-cms
# [5.0] New task plugin for check in items

New feature

Thanks to @brianteeman for the idea.

### Summary
New task plugin to check in all checked  out items. 

### Testing Instructions
* Apply patch
* Check out some items
* Create a new task in system/Scheduled Tasks
* Select Global Checkin
* Set Execution Rule to Manual Execution
* Set a Title
* Save
* You should see something like this

<img width="1348" alt="grafik" src="https://user-images.githubusercontent.com/467356/169640897-cebb8a65-6e1e-4bb7-ad1a-784525dd1e5e.png">

* Click on Run Task
* Go Back to the checked out items, they should be checked in

You can also set a delay time in hours, then only items that are checkout before actual - delay time will be checked in. 


### Documentation Changes Required
Documentation needs to be updated
